app/src/main/java/org/simple/clinic/illustration/HomescreenIllustrationModule.kt

```diff
@@ -1,12 +1,26 @@
 package org.simple.clinic.illustration
 
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.Types
 import dagger.Module
 import dagger.Provides
 import org.simple.clinic.AppDatabase
+import org.simple.clinic.remoteconfig.ConfigReader
 
 @Module
 class HomescreenIllustrationModule {
 
   @Provides
   fun illustrationDao(appDatabase: AppDatabase) = appDatabase.illustrationDao()
+
+  @Provides
+  fun illustrations(
```

Comment:
This will load and parse the illustrations on the UI thread. We should do this in the `HomescreenIllustrationRepository`, but load them from the config instead of the database.

Reply:
I would like to personally discuss with you on how we can do this.